### PR TITLE
Reformatted all XML files as a start point before using the auto-xmlformat GH action

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

**Comment by tbazant** [comment]
@lproven do not proofread this one, it's only a xmlformatting thing

**Review by asettle — APPROVED** [review]

File: xml/admin_about.xml [hunk 1]
```diff
@@ -217,9 +217,9 @@
    <title>&ceph; Nodes and Daemons</title>
    <para>
     In &ceph;, nodes are servers working for the cluster. They can run several
-    different types of daemons. We recommend to run only one type of
-    daemon on each node, except for &mgr; daemons which can be collocated with
-    &mon;s. Each cluster requires at least &mon;, &mgr;, and &osd; daemons:
+    different types of daemons. We recommend to run only one type of daemon on
```
Comment (lproven) [inline xml/admin_about.xml:220]: "We recommend running"
-- this is a common German-to-English grammatical error

File: xml/admin_about.xml [hunk 2]
```diff
@@ -217,9 +217,9 @@
...
+    each node, except for &mgr; daemons which can be collocated with &mon;s.
```
Comment (lproven) [inline xml/admin_about.xml:221]: > co-located

File: xml/admin_ceph_datamgm.xml [hunk 3]
```diff
@@ -1440,88 +1440,96 @@ osdmap e13 pg 1.6c (1.6c) -> up [1,0] acting [1,0]
...
+   </variablelist>
+   <note>
+    <para>
+     We recommend using the <emphasis>on</emphasis> mode, but administrators
+     who want to exercise more manual control should use the
+     <emphasis>warn</emphasis> mode.
+    </para>
+   </note>
+   <para>
+    To enable the autoscale for a particular pool:
```
Comment (lproven) [inline xml/admin_ceph_datamgm.xml:1524]: Either "to enable autoscaling" or "to enable the autoscaler", I think.

**Pull request merged**